### PR TITLE
Release wdio-wikibase

### PR DESCRIPTION
**Pull request by dati18**
To remove Dependabot's message ` Dependabot cannot update glob-parent to a non-vulnerable version`. I updated glob-parent before I can release.

### CODE REVIEW

**Review by toban — COMMENTED** [review]
I think we need the changes made to package.json too

**Comment by toban** [comment]
> I think we need the changes made to package.json too



> I think we need the changes made to package.json too

No I'm saying the wrong thing it seems to be in the package-lock.json file only

**Comment by dati18** [comment]
> > I think we need the changes made to package.json too
> 
> > I think we need the changes made to package.json too
> 
> No I'm saying the wrong thing it seems to be in the package-lock.json file only

Yes It only needs to be updated in package-lock.json, if you want it to be installed then the changes will be made to package.json

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	"requires": true,
 	"packages": {
 		"": {
+			"name": "wdio-wikibase",
```
Comment (toban) [inline package-lock.json:8]: I wonder why it's adding this but does the same thing for me

**Review by toban — APPROVED** [review]

**Pull request merged**